modules/ROOT/pages/subqueries/subqueries-in-transactions.adoc

```diff
@@ -550,3 +550,83 @@ RETURN largeLists
 1+d|Rows: 2
 
 |===
+
+[[composite-databases]]
+== Composite databases
+`CALL { ... } IN TRANSACTIONS` works on composite databases.
```

Comment:
```suggestion
`CALL { ... } IN TRANSACTIONS` can be used on composite databases.
```

Reply:
should we link out to documentation on composite databases here?

Reply:
Yes, I guess we should link composite doc